test/libsolidity/SolidityExecutionFramework.cpp

```diff
@@ -86,5 +87,8 @@ bytes SolidityExecutionFramework::compileContract(
 	else
 		obj = m_compiler.object(_contractName.empty() ? m_compiler.lastContractName() : _contractName);
 	BOOST_REQUIRE(obj.linkReferences.empty());
+	if (m_showMetadata)
+		cout << "metadata: " <<
+			m_compiler.metadata(_contractName.empty() ? m_compiler.lastContractName() : _contractName) << endl;
```

Comment:
Well, the coding style actually says "either break everything or nothing" :-D. But I might have a nice suggestion to get around it here: there's three times ``_contractName.empty() ? m_compiler.lastContractName() : _contractName`` in the function, so just make it a local variables and this should fit in one line anyways.